README.rst

```diff
@@ -364,6 +368,36 @@ Synapse can be installed via FreeBSD Ports or Packages contributed by Brendan Mo
  - Ports: ``cd /usr/ports/net/py-matrix-synapse && make install clean``
  - Packages: ``pkg install py27-matrix-synapse``
 
+
+OpenBSD
+-------
+
+There is currently no port for OpenBSD. Additionally, OpenBSD's security
+settings require a slightly more difficult installation process.
+
+1) Ensure that you have the Bourne Again Shell (``bash``) installed. The Korn
+   Shell (``ksh``), which is OpenBSD's default shell, does not support the
+   ``source`` command necessary to activate the python virtualenv to be set up
+   later.
```

Comment:
ksh does support "sourcing" files - it just uses the `.` syntax: `. bin/activate` for example.